assets/queries/cloudFormation/security_groups_not_vpc/query.rego

```diff
@@ -0,0 +1,44 @@
+package Cx
+
+CxPolicy [ result ] {
+   document := input.document
+   resources := document[i].Resources[name]
+   not_exists_vpc := check_exists_vpc(resources)
```

Comment:
To improve readability, make *check_exists_vpc* return true when vpc **does** exist and apply the not operator outside.